frontend/src/js/tooltip/AdditionalInfoHoverable.tsx

```diff
@@ -27,18 +27,24 @@ const AdditionalInfoHoverable = ({
   node,
   className,
   children,
+  parent,
 }: {
   children: ReactNode;
   node: ConceptT;
   className?: string;
+  parent?: string;
 }) => {
   const dispatch = useDispatch();
 
   const onDisplayAdditionalInfos = () => {
     if (!node.additionalInfos && isEmpty(node.matchingEntries)) return;
-
     dispatch(
-      displayAdditionalInfos({ additionalInfos: getAdditionalInfos(node) }),
+      displayAdditionalInfos({
+        additionalInfos: {
+          ...getAdditionalInfos(node),
+          parent,
```

Comment:
Now adding it outside of the `getAdditionalInfos` function is a little confusing when `getAdditionalInfos` returns an `AdditionalInfosType` which includes `parent?: string`, no?